acceptance/tests/augeas_validation.rb

```diff
@@ -0,0 +1,42 @@
+skip_test 'non-windows only test' if hosts.any? { |host| host.platform =~ /windows/ }
+tag 'audit:high'
+test_name 'Augeas Validation' do
+  teardown do
+    hosts.each do |agent|
+      file = <<-EOF
+augeas { 'test_ssh':
+  lens    => 'Ssh.lns',
+  incl    => '/etc/ssh/ssh_config',
+  context => '/files/etc/ssh/ssh_config',
+  changes => [
+    'remove Host testing.testville.nil'
+  ]
+}
+EOF
+      create_remote_file(agent, '/tmp/augeas_validation_teardown.pp', file)
+      on(agent, '/opt/puppetlabs/puppet/bin/puppet apply /tmp/augeas_validation_teardown.pp')
```

Comment:
Oh, and do we normally fully qualify the `puppet` path? Seems fine to just assume it is installed and available?

Reply:
I think the way we setup our acceptance tests a pre-suite does set the PATH to include this, so yeh it probably isn't needed. 
The habit comes from working on PE acceptance_tests where we didn't muck with the PATH as part of the pre-suite, and I think maybe at one point there was a bug with SPARC or AIX didn't get symlinks created to some of the Puppet binaries.